tests/subsys/pm/power_mgmt/src/main.c

```diff
@@ -444,6 +452,36 @@ ZTEST(power_management_1cpu, test_device_state_lock)
 	testing_device_lock = false;
 }
 
+ZTEST(power_management_1cpu, test_empty_states)
+{
+	const struct pm_state_info *cpu_states;
+	uint8_t state = pm_state_cpu_get_all(1u, &cpu_states);
+
+	zassert_equal(state, 0, NULL);
+}
+
+ZTEST(power_management_1cpu, test_force_state)
+{
+	forced_state = PM_STATE_STANDBY;
+	bool ret = pm_state_force(0, &(struct pm_state_info) {forced_state, 0, 0});
+
+	zassert_equal(ret, true, "Error in force state");
+
+	testing_force_state = true;
+	k_sleep(K_SECONDS(1U));
+}
+
+
+ZTEST(power_management_1cpu, test_state_next_get)
+{
+	static const struct pm_state_info state = {.state = PM_STATE_ACTIVE};
+	const struct pm_state_info *next;
+
+	next  = pm_state_next_get(0);
+	zassert_equal(next->state, state.state, NULL);
+}
+
```

Comment:
This test is not really helpful. Would be nice to test some particular state selected by the policy. Unfortunately this test does not declare any power state.
Reviewing this pr I noticed that this API is not reflecting a forced power state, I am fixing it and will add a test to ensure that this works correctly.

Drop this test only please, the rest looks good to me.